forms/CommentCreateForm.php

```diff
@@ -67,9 +70,11 @@ public function save()
         $Comment = $this->Comment;
 
         if (empty($this->id)) {
-            $Comment = new Comments\models\Comment();
+            $Comment =  \Yii::$app->getModule(Comments\Module::NAME)->model('comment');
```

Comment:
Your solution does not allows you to configure object in the declaration. Why initialize the module to use a single model?

To define dependencies have a special bootstrap configuration. Example yii2-app-advanced
[/common/config/bootstrap.php](https://github.com/yiisoft/yii2-app-advanced/blob/master/common/config/bootstrap.php)

I think to simplify initialization module, you can use your solution. Still I think it's the wrong approach. DI container that it exists. It is necessary to show people how to work with the right design patterns.